include/ck/utility/amd_buffer_addressing.hpp

```diff
@@ -1362,4 +1362,33 @@ amd_buffer_atomic_max(const typename vector_type_maker<T, N>::type::type src_thr
 #endif
 }
 
+// Direct loads from global to LDS.
+__device__ void
+llvm_amdgcn_raw_buffer_load_lds(int32x4_t rsrc,
+                                __attribute__((address_space(3))) float* lds_ptr,
+                                index_t size,
+                                index_t voffset,
+                                index_t soffset,
+                                index_t offset,
+                                index_t aux) __asm("llvm.amdgcn.raw.buffer.load.lds");
+
+template <typename T>
+__device__ void amd_direct_load_global_to_lds(const T* global_base_ptr,
```

Comment:
Do we need to add padding support?

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/ca7c6107623caed2743fdc6dc1c5d62749e33462/include/ck/utility/amd_buffer_addressing.hpp#L799-L810

Reply:
Used the "OOB offset" approach for the padded cases.